src/mdbExtensionController.ts

```diff
@@ -156,12 +156,14 @@ export default class MDBExtensionController implements vscode.Disposable {
     // Register our extension's commands. These are the event handlers and
     // control the functionality of our extension.
     // ------ CONNECTION ------ //
-    this.registerCommand(EXTENSION_COMMANDS.MDB_OPEN_OVERVIEW_PAGE, () =>
-      this._webviewController.openWebview(this._context)
-    );
-    this.registerCommand(EXTENSION_COMMANDS.MDB_CONNECT, () =>
-      this._webviewController.openWebview(this._context)
-    );
+    this.registerCommand(EXTENSION_COMMANDS.MDB_OPEN_OVERVIEW_PAGE, () => {
+      this._webviewController.openWebview(this._context);
+      return Promise.resolve(true);
```

Comment:
Right now we have the `registerCommand` interface expect a boolean always. I don't think we need to be so prescriptive there and it would be good to get rid of that strict extra typing. I'm thinking it would be better in another refactor pr though.